internal/services/mssql/mssql_database_resource_test.go

```diff
@@ -96,7 +125,7 @@ func TestAccMsSqlDatabase_complete(t *testing.T) {
 				check.That(data.ResourceName).Key("tags.ENV").HasValue("Test"),
 			),
 		},
-		data.ImportStep("sample_name"),
+		data.ImportStep("sample_name", "max_size_gb"),
```

Comment:
we shouldn't be adding `max_size_gb` to data.ImportStep on any of the tests - could you have another look at this change to get these passing without the need for this?

Reply:
Hi @catriona-m thanks for your time and feedback. Adding `max_size_gb` to data.ImportStep is because the following reason. Could you please give any other suggestions for this situation?
![image](https://github.com/user-attachments/assets/8d65fc60-44ec-44c9-95e2-dff83dbabaf8)

Reply:
Hi @catriona-m thanks for your feedback. I have updated the code. Could you please take another look?